python/simulation.cpp

```diff
@@ -33,9 +33,9 @@ void register_simulation(pybind11::module& m) {
         .def("run", &arb::simulation::run,
             pybind11::call_guard<pybind11::gil_scoped_release>(),
             "Run the simulation from current simulation time to tfinal, with maximum time step size dt.",
-            "tfinal"_a, "dt"_a)
+            "tfinal"_a, "dt"_a=0.025)
```

Comment:
Maybe add default and units (of dt and tfinal) to description?

Reply:
I have added the unit here, however these further changes are beyond the scope of this PR.